app/templates/base.html

```diff
@@ -0,0 +1,15 @@
+<html>
+<head>
+    {% if title %}
+    <title>{{ title }} - Web-Hook</title>
+    {% else %}
+    <title>DRC Web-Hook</title>
+    {% endif %}
+</head>
+
+<body>
+    <div>DRC Web-Hook: <a href="/">Home</a></div>
```

Comment:
What if this is not for DRC? The `title` above seems to support others but this `div` does not.